### PR TITLE
Add passive review prompt

### LINKED ISSUE #1295: Passive review prompt
A passive prompt somewhere inside the new settings page for very active users, suggesting to leave a review for the extension. This way it will only be seen if the settings page is being visited. It will not involve any kind of notification.

The prompt can be dismissed, e.g. "Never ask again".

**Plans:**
- Detect "Active user" in the settings page. We'll probably need to start ticking a storage value for active days, or entries created in button etc.
- Show the passive prompt. Perhaps it can go in the footer? It's even less intrusive there than in the main body of any settings tabs.
- The prompt must open the correct web extension store for the browser being used.
  - If it is opened, never show it again.
  - If it is **dismissed**, never show it again.
- If user uses the **Reset all settings** feature, **DO NOT** reset the sync storage value for this feature.

### PR DESCRIPTION
**Pull request by shantanuraj**
Please remember the [Contributing Guidelines](https://github.com/toggl/toggl-button/blob/master/.github/CONTRIBUTING.md) :heart:

## :star2: What does this PR do?

- Add a review prompt on the settings page
- Show review prompt on opening settings for _active users_

## :bug: Recommendations for testing

All changes should be tested across Chrome and Firefox.

- Open settings
- In the console paste 
```javascript
localStorage.setItem('dismissedReviewPrompt', false);
localStorage.setItem('timeEntriesTracked', 30);
location.reload();
```
- It show show the review prompt
- Click the review button to land on the correct store for your browser
- Click on the close on the bottom right of the prompt to dismiss it

## :memo: Links to relevant issues or information

Closes #1295

### CODE REVIEW

File: src/scripts/lib/db.js [hunk 1]
```diff
@@ -34,7 +34,8 @@ const DEFAULT_SETTINGS = {
 };
 
 const LOCAL_ONLY = {
-  [ORIGINS_KEY]: true
+  [ORIGINS_KEY]: true,
+  seenReviewPrompt: true
```
Comment (tcrammond) [inline src/scripts/lib/db.js]: nitpick: this is more that it's been *dismissed*, rather than seen

File: src/html/settings.html [hunk 2]
```diff
@@ -314,13 +314,24 @@ <h2>Custom Integrations</h2>
...
+          <div>Version <span id="version"></span></div>
+          <div>
+            Changelog: <a href="http://toggl.github.io/toggl-button">http://toggl.github.io/toggl-button</a>
+          </div>
+        </div>
       </div>
     </div>
+    <div id="review-prompt">
+      <span>Enjoying Toggl Button?</span>
+      <a target="_blank" rel="noopener">
```
Comment (tcrammond) [inline src/html/settings.html]: I believe this is the "correct" attributes for security 🔒 

```suggestion
      <a target="_blank" rel="noopener noreferrer">
```

Comment (shantanuraj) [reply src/html/settings.html]: Done ✅

**Comment by shantanuraj** [comment]
addressed 👍

**Pull request merged**